brainstormforce/astra
# fix: Site Identity and Menu elements style issues

### Description
Fixed a few issues related to styling controls of Menu and Site Identity elements

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
- Go to - Header Builder > Site Identity > Mobile Logo options
- Go to - Header Builder > Primary Menu > Check if parent menu style options inherit for sub menu.

### Checklist:
